docs/csharp/misc/CS4009.md

```diff
@@ -39,6 +39,9 @@ public class Example
 
 ## To correct this error
 
+> [!NOTE]
+> Starting with C# 7.1, the `Main` method can have an `async` modifier. For more information, see the [Async main](../whats-new/csharp-7-1.md#async-main) section of the [What's new in C# 7.1](../whats-new/csharp-7-1.md) article. For the information about how to select C# language version, see the [Select the C# language version](../language-reference/configure-language-version.md) article.
```

Comment:
the information --> information
C# language version --> the C# language version